Docs: fixed descriptions (#3125)
juicedata/juicefs: docs/en/introduction/io_processing.md

sidebar_label: Data Processing Workflow
sidebar_position: 3
slug: /internals/io_processing

> description: This article introduces the read and write processes of JuiceFS respectively, and the process of implementing JuiceFS read-write chunking technology.

Reviewer: ```suggestion
description: This article introduces read and write implementation of JuiceFS, including how it split file into chunks.
```
